src/pages/Rockets.jsx

```diff
@@ -1,9 +1,55 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRockets, reserveRocketAction } from './RocketSlice';
+import './rockets.css';
```

Comment:
you  might need to update your component imports as shown below to make use of the react-bootstrapp library:

```
import React, { useEffect } from 'react';
import Badge from 'react-bootstrap/Badge';
import Button from 'react-bootstrap/Button';
import { useDispatch, useSelector } from 'react-redux';
import { fetchRockets, reserveRocketAction } from './RocketSlice';
import './rockets.css';
```